GeneralsMD/Code/GameEngine/Source/GameClient/ParabolicEase.cpp

```diff
@@ -59,7 +59,7 @@ ParabolicEase::setEaseTimes(Real easeInTime, Real easeOutTime)
 		m_out = clamp(m_out);
 	}
 
-	if (m_in > m_out) {
+	if (m_in > m_out + FLT_EPSILON) {
 		DEBUG_CRASH(("Ease-in and ease-out overlap (in = %g, out = %g)", m_in, m_out));
 		m_in = m_out;
 	}
```

Comment:
```cpp
if (m_in > m_out) {
		DEBUG_ASSERTCRASH(m_in <= m_out + FLT_EPSILON, ("Ease-in and ease-out overlap (in = %g, out = %g)", m_in, m_out));
		m_in = m_out;
}
```

I think, because we still want to set `m_in = m_out` on slight mismatch.